lib/rucio/tests/test_replica.py

```diff
@@ -501,6 +502,40 @@ def test_set_tombstone(self):
         with assert_raises(ReplicaNotFound):
             set_tombstone(rse_id, scope, name)
 
+    def test_list_replicas_with_updated_after(self):
+        """ REPLICA (CORE): Add and list file replicas with updated_after filter """
+        scope = InternalScope('mock')
+        root = InternalAccount('root')
+        mock = get_rse_id(rse='MOCK')
+        dsn = 'ds_ua_test_%s' % generate_uuid()
+        add_did(scope=scope, name=dsn, type='DATASET', account=root)
+        #
+        t0 = datetime.utcnow()
+        time.sleep(2)
```

Comment:
not really happy about the sleeps, but i don't see how this could be done otherwise